src/actions/markdownActions.test.js

```diff
@@ -0,0 +1,13 @@
+import { updateMarkdown, UPDATE_MARKDOWN } from './markdownActions';
+
+
+describe('test markdown actions', () => {
+  it('updates a markdown file', () => {
```

Comment:
this doesn't update a markdown file. It creates an action that can be used to update a markdown file